QuizAppNew/View/QuestionView.swift

```diff
@@ -0,0 +1,182 @@
+            Button(action: {}, label: {
+                Text(question.optionA)
+                    .foregroundColor(Color.black)
+                    .padding([.horizontal,.vertical],5)
+                    .frame(maxWidth: .infinity)
+                    .background(
+                        RoundedRectangle(cornerRadius: 15)
+                            .stroke(color(option: question.optionA), lineWidth: 1)
+                    )
+            })
+            
+            //FOR OPTIONS....B
+            
+    //        Button(action: {selected = question.optionB}, label: {\
+            Button(action: {}, label: {
+                Text(question.optionB)
+                    .foregroundColor(Color.black)
+                    .padding([.horizontal,.vertical],5)
+                    .frame(maxWidth: .infinity)
+                    .background(
+                        RoundedRectangle(cornerRadius: 15)
+                            .stroke(color(option: question.optionB), lineWidth: 1)
+                    )
+            })
+            
+            //FOR OPTIONS....C
+            
+        //    Button(action: {selected = question.optionC}, label: {
+            Button(action: {}, label: {
+                Text(question.optionC)
+                    .foregroundColor(Color.black)
+                    .padding([.horizontal,.vertical],5)
+                    .frame(maxWidth: .infinity)
+                    .background(
+                        RoundedRectangle(cornerRadius: 15)
+                            .stroke(color(option: question.optionC), lineWidth: 1)
+                    )
+            })
+            
+            //FOR OPTIONS....D
+            
+         //   Button(action: {selected = question.optionD}, label: {
+            Button(action: {}, label: {
+                Text(question.optionD)
+                    .foregroundColor(Color.black)
+                    .padding([.horizontal,.vertical],5)
+                    .frame(maxWidth: .infinity)
+                    .background(
+                        RoundedRectangle(cornerRadius: 15)
+                            .stroke(color(option: question.optionD), lineWidth: 1)
+                    )
+            })
+            
+            Spacer(minLength: 0)
+            
+            //Buttons use for submitted answer And Next Question use with HStack...
+            
+            HStack(spacing: 15) {
+                
+            //    Buttons use for submitted answer
+                
+                Button(action: checkAns, label: {
+                    Text("Submit")
+                        .fontWeight(.heavy)
+                        .foregroundColor(.white)
+                        .padding(.vertical)
+                        .frame(maxWidth: .infinity)
+                        .background(Color.blue)
+                        .cornerRadius(15)
+                })
+                
+                // disabling
```

Comment:
Remove all this logic. your response to making UI design not business logic

Reply:
All business logic is removed